contracts/Marketplace.sol

```diff
@@ -284,28 +282,18 @@ contract Marketplace is Collateral, Proofs {
     return _isCancelled(requestId);
   }
 
-  function _host(SlotId slotId) internal view returns (address) {
-    return slots[slotId].host;
-  }
-
   function getHost(SlotId slotId) public view returns (address) {
-    return _host(slotId);
+    return slots[slotId].host;
   }
 
-  function _request(
+  function getRequest(
     RequestId requestId
-  ) internal view returns (Request storage) {
+  ) public view returns (Request memory) {
```

Comment:
This creates a storage > memory copy, right? Any particular reason this is needed?

Reply:
I wanted to combine `_request()` and `getRequest()`, but since `getRequest` is public, it is not allowed to return a storage location. Now that you mention it, we should probably keep the two separate. I'll revert this change.